### PR TITLE
Zhyass monitor

### PR DESCRIPTION
**Pull request by zhyass**

### CODE REVIEW

File: src/backend/connection.go [hunk 1]
```diff
@@ -13,14 +13,16 @@ import (
 	"errors"
 	"fmt"
 	"io"
+	"monitor"
 	"sync"
 	"time"
 	"xbase/stats"
 
+	"xbase/sync2"
+
```
Comment (andyli029) [inline src/backend/connection.go]: monitor
xbase/stats

modify the place。

**Review by andyli029 — COMMENTED** [review]
two commit should be one commit.

File: src/backend/connection.go [hunk 2]
```diff
@@ -84,6 +86,7 @@ func NewConnection(log *xlog.Log, pool *Pool) Connection {
 func (c *connection) Dial() error {
 	var err error
 	defer mysqlStats.Record("conn.dial", time.Now())
+	defer monitor.BackendConnectionInc(c.address)
```
Comment (andyli029) [inline src/backend/connection.go:88]: if the conn is bad and closed, but it be added. please review again.

Comment (zhyass) [reply src/backend/connection.go:88]: if the conn is bad and closed,  then call close() ,the added variable will be decremented

Comment (andyli029) [reply src/backend/connection.go:88]: if the first conn, then it will be -1,

**Pull request closed without merging**